apps/nextjs-website/jest.config.ts

```diff
@@ -12,6 +13,13 @@ const jestConfig: JestConfigWithTsJest = {
     // Needed or else if some test files imports a file from the src folder using the Next naming convention, it will fail.
     // Example: import { shared } from '@/_contents/translations'; returns Cannot find module '@/_contents/translations' from 'src/some-file.ts'
     '@/(.*)': '<rootDir>/src/$1',
+    '^uuid$': require.resolve('uuid'),
```

Comment:
why the uuid here? what is the purpose of importing the `uuid` library?

Reply:
For the same reason as [above](https://github.com/pagopa/developer-portal/pull/581#discussion_r1479797176).